storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -2809,21 +2809,19 @@ public void launchServer() throws Exception {
                                         }
                                     });
 
+            //Be cautious using method reference instead of lambda. subexpression preceding :: will be evaluated only upon evaluation
```

Comment:
Nit: Not sure about this comment, it's not really specific to this code, but applies to any use of method references anywhere.

Reply:
Sorry I was writing this to myself because I was not super familiar with Java lambda. Should probably delete it as it's not relevant to code.